packages/block/__tests__/blockchainMachine.test.ts

```diff
@@ -19,9 +19,10 @@ import { BlockchainMachine } from "../src/blockchainMachine";
 import { throwingInstance, fnIt, wait } from "@pisa-research/test-utils";
 import { ArgumentError, ConfigurationError } from "@pisa-research/errors";
 import chaiAsPromised from "chai-as-promised";
+import { PlainObject, DbObject, defaultSerialiser } from "@pisa-research/utils";
 chai.use(chaiAsPromised);
 
-type TestAnchorState = { number: number; extraData: string };
+type TestAnchorState = { number: number; extraData: string } & PlainObject;
```

Comment:
Again, I dont think this & PlainObject is required, can you check all the cases of & PlainObject and see if they're necessary?

Reply:
It compiles without, so removed.